### ISSUE TITLE
[Portal 2] After epilogue FMV plays, the game crashes

### ISSUE THREAD

**Issue by Dogman15** [ValveSoftware/portal2#428]
It seems that a number of people, both reporting on Steam and as seen in Let's Plays on YouTube, are encountering a bug where after "sp_ending_callback.bik" plays (the cutscene of Wheatley and the Space Core orbiting him), the game either crashes, or it stays on a black screen instead of going back to the main menu (menu_act05.bik while the music "Space Phase" plays).

Some people have reported that the issue was fixed for them by:
1. playing on borderless fullscreen windowed, or just windowed
2. pausing the game during the final cutscene (which does not pause video playback) and bringing up the Steam overlay
3. disabling "wait for vertical sync"
4. not putting shader detail on "very high"
Or any combination of these, possibly all or just one. As outlined in some of the responses to this discussion topic on the Steam forums: https://steamcommunity.com/app/620/discussions/0/3044978964799136435/

One person has made a guide to help others: https://steamcommunity.com/sharedfiles/filedetails/?id=3014856840

However, I have tried all of these things, and the game would still crash for me after the Wheatley/Space Core cutscene. This happened both when I would load up "map sp_a5_credits" and wait, and also if I started a "new game" from the main menu.

Here is one video I could find where I have seen this happen:
https://www.youtube.com/watch?v=trrIJuZt3Kg (1:05:08)
Another (1:30:15 - after sp_ending_callback, the player does not show the menu at all, implying it didn't appear): https://www.youtube.com/watch?v=nVsj0QOIt28

**Comment by kisak-valve** [comment]
sp_a5_credits no longer boots to the menu at the end of the credits

Issue transferred from https://github.com/ValveSoftware/portal2/issues/432.
@vrad-exe posted on 2024-05-09T02:32:00:

When playing the credits from the New Game menu (not after completing finale4 normally), it now traps the player in a black box when the credits finish instead of booting to the menu as it did previously. The console indicates that Portal 2 now seemingly enforces the `FCVAR_CLIENTCMD_CAN_EXECUTE` convar flag since a recent update, but the credits weren't changed to call `ClientCmd_Unrestricted` when disconnecting the player so it now fails.

**Comment by NathanL1370** [comment]
Another detail I want to add is that the `hud_subtitles` command is also affected by the `FCVAR_CLIENTCMD_CAN_EXECUTE` flag, showing no subtitles when the credits start. I have captions enabled and this is very jarring to notice.

Exiting to the main menu after the callback scene finishes also crashes the game.

**Comment by MeowcaTheoRange** [comment]
Can confirm, happens to me on Steam Deck and Ultramarine Linux 39

**Comment by Loughty** [comment]
Happens to me, i'm using Ubuntu 24.04 LTS and the lasted version of Portal 2

Hardware:
- Dell Inc. XPS 8920
- Intel Core i7-7700
- 8GB of RAM
- Intel HD Graphics 630 (KBL GT2)

Software:
- The default DE, Gnome Shell
- Wayland
- Linux 6.8.0-36-generic

Here some videos:

[Grabación de pantalla desde 2024-07-11 16-56-21.webm](https://github.com/user-attachments/assets/afc7fc15-61b4-4f4b-8196-81aab4376f9b)
(widowed)

[Grabación de pantalla desde 2024-07-11 17-24-49.webm](https://github.com/user-attachments/assets/3a085aef-ae50-4597-9a66-d8a5ad577f0d)
(widowed + pausing + disabling vsync + lowering shader detail to "high")

A GIF because Github don't show a preview:
![Grabacindepantalladesde2024-07-1116-56-21-ezgif com-optimize](https://github.com/user-attachments/assets/bff4c895-d824-4452-91dd-11bbe313d9e8)

I hope this helps

**Comment by Loughty** [comment]
I manage to fix it, but I do many things at the same time, so i really don't know what do the trick, but for the record:

First, I go to Steam Properties > Betas, and select "previous_release".

Then, go to graphics:
- Widowed + no sync
- All graphics settings to low, including "none" on anti-aliasing and texture filter
- Multicore rendering deactivated

Then... to audio:
- 2 speakers
- Sound quality low
- No subtitles
- No puzzle creator sound

I think is the "previous release" thing who do the real fix, gives me the compilation ID "6318595" ("14822445" is the current has this date), but if you still have the problem, then use the same config that I use.

![Captura desde 2024-07-11 17-55-27](https://github.com/user-attachments/assets/8ecc47d3-5aac-49ee-89ac-dc6c1d63a883)

Here a guide that I made, in Spanish:
https://steamcommunity.com/sharedfiles/filedetails/?id=3287191992

**Comment by NathanL1370** [comment]
> Replying to https://github.com/ValveSoftware/portal2/issues/428#issuecomment-2224032912

For me, I didn't have to change any settings when the credits finished on the "previous_release" build. It just kicked me back to the menu.

**Comment by Hardingfele2212** [comment]
I'm on Windows 10.
When the cutscene finishes playing it all turns black. However, it does not "trap" the game—pulling up the mid-game menu and disconnecting to the main menu works just fine, and even gives the space background as if the game finished normally. Sort of like older games which forced you to reset or exit to truly "finish" them, though it's not intentional of course.

![Image](https://github.com/user-attachments/assets/f855f26a-d7e6-475d-810f-77a951ff963e)
![Image](https://github.com/user-attachments/assets/15f04b4d-2578-4654-bd84-3a7b91b7a643)

**Comment by vrad-exe** [comment]
Yeah, I believe that's a separate issue from this one, but [when I reported it](https://github.com/ValveSoftware/portal2/issues/432) it was closed as a duplicate. 🤷‍♀

**Comment by manlaya** [comment]
Same issue here : back screen after Whitney say "the end", crash if I manually go back to main screen.

I did try the different tricks (video/audio settings) and it did not work in my case. 
And, the `previous_release` do not launch...

System Details Report

Hardware Information:
- **Hardware Model:**                        Micro-Star International Co., Ltd. Katana 15 B12VGK
- **Memory:**                                      32.0 GiB
- **Processor:**                                   12th Gen Intel® Core™ i7-12650H × 16
- **Graphics 1:**                                  NVIDIA GeForce RTX™ 4070 Laptop GPU

Software Information:
- **OS Name:**                                     Fedora Linux 42 (Workstation Edition)
- **GNOME Version:**                            48
- **Windowing System:**                      Wayland
- **Kernel Version:**                              Linux 6.18.5-100.fc42.x86_64

**Comment by vrad-exe** [comment]
Fixed as of the latest update on windows, might be good to have someone retest on linux

**Comment by manlaya** [comment]
Hi,
Running fedora linux 43, the latest update fixed it too for me ! Thanks !

**Comment by vrad-exe** [comment]
Can confirm that this is also fixed for me on linux.
